markdown/ranger/ranger-integration-config.html.md.erb

```diff
@@ -129,6 +129,64 @@ Once the connection between HAWQ and Ranger is configured, you may choose to set
 5. Click **Save** to save your changes.
 6. Select **Service Actions > Restart All** and confirm that you want to restart the HAWQ cluster.
 
+## <a id="enableha"></a>Step 3: (Optional) Register a Standby Ranger Plug-in Service
+
+The HAWQ Ranger Plug-in Service runs on the HAWQ master node. If this service goes down, all HAWQ database operations will fail. Configure a highly available HAWQ Ranger Plug-in Service to eliminate possible downtime should this situation occur.
+
+Ranger Admin high availability and HAWQ Ranger Plug-in Service high availability are independent; you can configure HAWQ Ranger Plug-in Service HA without configuring HA for Ranger Admin.
```

Comment:
Might want to add that configuring HA for both is advised?  Also, this section uses the abbreviated term "Ranger Admin" throughout, which seems confusing to me.  Should probably stick with "Ranger Administration Host" to stay consistent with the first part of the doc.